corehq/apps/sms/migrations/0001_initial.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import corehq.apps.sms.models
+import dimagi.utils.couch.migration
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsforms', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='MessageLogOld',
```

Comment:
I think this should be safe to delete from the migration. It's not used in south_migrations and on staging/prod it looks like this table has a different owner and hasn't been written to in 3.5 years.

It also isn't created on new instances since neither `syncdb` or `migrate` will update sms

Is there any reason it's still around in `models.py` ? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/models.py#L525-L552

Tagging last modifiers: @dannyroberts @czue @gcapalbo